recipe/meta.yaml

```diff
@@ -13,10 +13,12 @@ source:
   {{ hash_type }}: {{ hash }}
 
 build:
-  number: 0
+  number: 1
   script: 
     - python setup.py install --record=record.txt  # [not py3k]
     - python setup3.py install --record=record.txt  # [py3k]
+    - copy %PREFIX%\Lib\site-packages\pywin32_system32\*.dll %PREFIX%
```

Comment:
Yeah, I hope it looks on PATH.  If not, I'd prefer to put DLLs in the site-packages folder with pywintypes.  Please avoid putting DLLs in the root prefix.  Let me know if you need help patching if necessary.